### ISSUE TITLE
Won't load with some metadata files - possibly file size related

### ISSUE THREAD

**Issue by msheehan79** [mmatyas/pegasus-frontend#633]
Have noticed Pegasus will not start at all when I try to load some metadata files. Seems like it might be a metadata file size issue - it displays the loading bar, then after a few seconds it exits. I can usually see the game count for brief second before it exists. The log file does not display anything unusual.

This was confirmed with build pegasus-fe_alpha13-89-g75ab136_win-mingw-static. Prior to this I was using pegasus-fe_alpha13-55-gb984bad_win-mingw-static and no issues.

- Win 10 x64
- Portable mode
- Metadata files located in the config/metafiles directory

I noticed that metadata files above 1MB seem to cause the crash, while files <1MB load file. Of the supplied files, the gbc loads fine, but the genesis one will crash.
[system-genesis.metadata.txt](https://github.com/mmatyas/pegasus-frontend/files/4603400/system-genesis.metadata.txt)
[system-gbc.metadata.txt](https://github.com/mmatyas/pegasus-frontend/files/4603401/system-gbc.metadata.txt)

**Comment by mmatyas** [comment]
Hm the file itself is fine, loads without problems on Linux. Will try later on Windows too, though the loading itself should work the same there too. I wonder maybe it's one of the assets that causes problems?

**Comment by msheehan79** [comment]
@mmatyas The problem file I included is just one of many that seem to trigger it, so I am thinking its unlikely to be an asset. This was a file that loaded just fine with the prior build I noted too.

I relabeled the entire artwork directory so none of them were found and the issue persists so unless the name of the file is enough to cause the crash I don't think it is tied to the asset.

Happy to try anything else you can think to help isolate this a bit more.

**Comment by mmatyas** [comment]
Actually I think I just managed to reproduce it; I was testing #631 and this just happened to me too right now. It might be related to missing files actually, will look into it.

**Comment by mmatyas** [comment]
Whoops, wrong issue. Meant to write this for #632.

**Comment by msheehan79** [comment]
@mmatyas Sounds good. And this is apparently not related to file size - I just wiped my config and started from scratch in portable mode, and reproduced it with this file which is only 500kb.
[system-saturn.metadata.txt](https://github.com/mmatyas/pegasus-frontend/files/4604100/system-saturn.metadata.txt)

**Comment by msheehan79** [comment]
@mmatyas Another bit of information which might be useful - I normally do not use other sources so I had all other data sources disabled in my config. 

I noticed if I use the default config with the default data sources enabled and just the saturn metadata file above it crashes on start. 

With all of the data sources disabled it does load, although when I moved the cursor around after just a few games it crashes with the error on #632 so I guess it can occur when switching games too not just collections.
